apps/web-giddh/src/app/shared/header/components/group-update/group-update.component.ts

```diff
@@ -514,11 +520,20 @@ export class GroupUpdateComponent implements OnInit, OnDestroy, AfterViewInit {
         this.destroyed$.complete();
     }
 
-
-
-
-
-
-
+    /**
+    * To get sub group belongs to mentioned group (currentliabilities , currentassets)
+    *
+    * @param {*} item
+    * @memberof GroupUpdateComponent
+    */
+    public isDebtorCreditorGroup(itemUniqueName: any): boolean {
+        let isTaxableGroup: boolean = false;
+        this.groupWithParentLessDetailsList.forEach(element => {
+            if (itemUniqueName === element.uniqueName) {
+                isTaxableGroup = element.parentGroups.some(groupName => groupName.uniqueName === 'sundrydebtors' || groupName.uniqueName === 'sundrycreditors');
```

Comment:
Please check if this will work when a single account is shared

Reply:
Yes working have checked already